locale/en-us/wifi_intro_2.dialog

```diff
@@ -1 +1 @@
-I have created a temporary WiFi network called OVOS, Connect to it from your phone or computer and follow the prompts, then I'll walk you through the rest
+I have created a temporary WiFi network called {{ssid}}, Connect to it from your phone or computer and follow the prompts, then I'll walk you through the rest
```

Comment:
Flip side of what I wrote above about the SSID itself: should we ensure the default is pronounceable by most TTS engines?

Reply:
I think the SSID should definitely be pronouncable by default.. If we implement some randomization, it should be a combination of words (a lot of ISP routers do this for default passwords now)